packages/cli/src/commands/list.ts

```diff
@@ -317,27 +357,67 @@ export default async function main(client: Client) {
   }
 }
 
-function getProjectName(d: Deployment) {
-  // We group both file and files into a single project
-  if (d.name === 'file') {
-    return 'files';
+function getDeploymentIdFromUrl(
+  url: string,
+  projectName: string | undefined,
+  inspect?: boolean
+): string {
+  if (!projectName) {
+    return url;
   }
+  return inspect
+    ? url.split('/').slice(-1)[0]
+    : url
+        .match(
+          /(?<=https:\/\/)(.*)(?=(.*)(\.vercel\.app|\.now\.sh|\.vercel\.sh))/g
+        )?.[0]
+        ?.split(`${projectName}-`)?.[1]
+        ?.split('-')?.[0] || url;
+}
 
-  return d.name;
+function getDeployUrl(
+  deployment: Deployment,
+  projectName: string | undefined,
+  inspect?: boolean
+): string {
+  const url = inspect ? deployment.inspectorUrl : 'https://' + deployment.url;
+  if (terminalLink.isSupported) {
+    const depId = getDeploymentIdFromUrl(url, projectName, inspect);
+    return terminalLink(depId, url);
```

Comment:
Agh, yeah I think since that was the takeaway from our team discussion, I'll remove `terminal-link` for now. BUT for the record I think `vc ls` is a great use case for using `terminal-link`. I mean, does this not look way better?

<img width="721" alt="181114770-ca947618-e0af-4b20-a387-b06552018b98" src="https://user-images.githubusercontent.com/14811170/184235352-e1d0759b-73b0-4756-9a46-16ba70094962.png">

Reply:
It does, but I think the user expectation issue is a valid one.